plugin.cpp

```diff
@@ -118,7 +118,7 @@ PLUGIN_HANDLE plugin_init(ConfigCategory* config,
 			}
 			else if (itr->value.IsNumber())
 			{
-				DatapointValue dpv((long) itr->value.GetInt());
+				DatapointValue dpv((long) itr->value.GetInt64());
```

Comment:
Strictly speaking we should be checking for IsUint64() as IsNumber() will return true for floating point values.

Reply:
> Strictly speaking we should be checking for IsUint64() as IsNumber() will return true for floating point values.

@MarkRiddoch updated checking to IsUint64() instead of IsNumber()